pkg/kubelet/reason_cache.go

```diff
@@ -40,8 +40,8 @@ type ReasonCache struct {
 	cache *lru.Cache
 }
 
-// Reason is the cached item in ReasonCache
-type reasonItem struct {
+// ReasonItem is the cached item in ReasonCache
+type ReasonItem struct {
```

Comment:
let keep this as private :)

Reply:
exported method Get returns unexported type *kubelet.reasonItem on line 97
```go
func (c *ReasonCache) Get(uid types.UID, name string) (*reasonItem, bool)
```
I think if I need to change reasonItem to exported?